### DISCUSSION TITLE
What happens to my contribution graph if I'm removed as a collaborator from a GitHub repository?

### DISCUSSION THREAD

**Discussion by kalehege** [community/community#143588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/48fe8960-9788-4aae-a9f0-0c550c09de97)

what happens for this, please tell me

**Comment by DylanDevelops** [comment, marked as answer]
Hi @kalehege! If your access to a repository is removed, your commits will still appear as contributions in your graph. It works similarly to how you can still see contributions even if you contribute to a private repo.

I hope this helps!
